src/main.ts

```diff
@@ -77,4 +77,33 @@ export class Minifier {
       }
     }
   }
+
+  private nextChar(c: string): string { return String.fromCharCode(c.charCodeAt(0) + 1); }
+
+  // Given the last code, returns a string for the new property name.
+  // ie: given 'a', will return 'b', given 'az', will return 'ba', etc. ...
+  generateNextLateralPropertyName(code: string): string {
```

Comment:
add support for '$' and '_', and exclude keywords (like 'if' and 'in')

https://mathiasbynens.be/notes/javascript-identifiers is a good read on what makes a valid js identifier. I suggest we ignore unicode for now.